cmd/globals.go

```diff
@@ -109,6 +109,9 @@ var (
 	// Time when object layer was initialized on start up.
 	globalBootTime time.Time
 
+	globalActiveCred       credential
+	globalPublicCerts      []*x509.Certificate
+	globalObjCacheDisabled bool
```

Comment:
> We would need to validate how multi-node minio is started. This check needs to be done in post login (currently not).

_MINIO_CACHE is an undocumented env-var mainly to be used by developers for testing erasure code (i.e so that we don't actually hit the cache and ignore the underlying problem)

Even otherwise _MINIO_CACHE is an minio process specific setting and hence need not be compared with how the peers are run with. i.e I should be able to run one server with caching and one without caching.